app/piwik.js

```diff
@@ -1,11 +1 @@
-var _paq = _paq || [];
-_paq.push(['setDomains', ['*.www.targetvalidation.org']]);
-// _paq.push(['trackPageView']);
-_paq.push(['enableLinkTracking']);
-(function () {
-    var u = '//opentargets.piwikpro.com/';
-    _paq.push(['setTrackerUrl', u + 'piwik.php']);
-    _paq.push(['setSiteId', 1]);
-    var d = document, g = d.createElement('script'), s = d.getElementsByTagName('script')[0];
-    g.type = 'text/javascript'; g.async = true; g.defer = true; g.src = u + 'piwik.js'; s.parentNode.insertBefore(g, s);
-})();
+/** overwrite this file with your own matomo/piwik .js */
```

Comment:
We need to keep the previous file here in the master branch. With either an existing or empty file, the behaviour for implementers is the same - replace the file.

Reply:
The problem is that when one does not replace the file, or perhaps makes a mistake or typo in the path, the default behavior of the installation is to report things back to the Open Targets Piwik/Matomo...One of the main goals of this PR is really to remove this default behavior.